stl/inc/xutility

```diff
@@ -1513,69 +1548,78 @@ _NODISCARD constexpr _Ty* end(_Ty (&_Array)[_Size]) noexcept {
 }
 
 template <class _Container>
-_NODISCARD constexpr auto cbegin(const _Container& _Cont) noexcept(noexcept(_STD begin(_Cont)))
+_NODISCARD constexpr auto cbegin(const _Container& _Cont) noexcept(noexcept(_STD begin(_Cont))) /* strengthened */
     -> decltype(_STD begin(_Cont)) {
     return _STD begin(_Cont);
 }
 
 template <class _Container>
-_NODISCARD constexpr auto cend(const _Container& _Cont) noexcept(noexcept(_STD end(_Cont)))
+_NODISCARD constexpr auto cend(const _Container& _Cont) noexcept(noexcept(_STD end(_Cont))) /* strengthened */

```

Comment:
Not strengthened: this is canon ([iterator.range]/7 ish)

Please don't ask me why _only_ `cbegin` and `cend` are conditionally `noexcept` in [iterator.range] and especially don't ask me to explain why they depend on the `noexcept`-ness of `std::begin` and `std::end` which are never `noexcept` - OMG this is a _dumpster fire_.